package.json

```diff
@@ -6,8 +6,9 @@
   "scripts": {
     "dev": "razzle start",
     "build": "razzle build",
-    "lint": "eslint --ext .js,jsx ./src",
-    "test": "razzle test --env=jsdom",
+    "test": "npm run test:lint & npm run test:unit",
```

Comment:
Use 2 ampersands rather than one, a single ampersand could be slightly confusing as it looks like it might run as a background process.